_posts/2025-02-27-cockpit-334.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Cockpit 334
+author: mvo
+date: '2025-02-27'
+tags: files
+slug: cockpit-334
+category: release
+summary: ''
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).
+We release regularly.
+
+Here are the release notes from cockpit-files 17:
+
+
+## Files: Apple-compatible keyboard shortcuts
+
+Cockpit Files now properly supports comannd-based keyboard shortcuts, also known by the `⌘` symbol, fitting in better with macOS and portable Apple devices using a paired keyboard.
```

Comment:
Whoops. I made a typo in the PR. I fixed it there, and here's a local suggested fix.

```suggestion
Cockpit Files now properly supports command-based keyboard shortcuts, also known by the `⌘` symbol, fitting in better with macOS and portable Apple devices using a paired keyboard.
```